SRTNet.h

```diff
@@ -315,12 +333,18 @@ class SRTNet {
     /**
      * @brief Server worker thread function when server accepts multiple clients, otherwise
      * used as a normal function for accepting one single client connection.
+     * @param singleSender If set to true, the function will exit and close the server socket once the first
+     * sender/client has successfully connected to the server, if set to false the function will keep on accepting
+     * incoming connections until the server is stopped.
+     * @return true if singleSender is true and a sender/client has successfully connected, false otherwise.
      */
     bool waitForSRTClient(bool singleSender);
```

Comment:
Now, thinking about it, this parameter should be called singleClient instead, since either the server or the client or both can be the sending side in a SRT connection.

Reply:
Yeah, I'll change that and also get rid of all sender/client comments :)

Reply:
https://github.com/Unit-X/srt-cpp/pull/15/commits/e7ab03bcee6785c7a00178fd618f3bd4678b6071